docs/user-guide/slicing.ipynb

```diff
@@ -6,10 +6,20 @@
    "source": [
     "# Slicing\n",
     "\n",
-    "Data in a [Variable](../generated/scipp.Variable.rst#scipp.Variable) or [Dataset](../generated/scipp.Dataset.rst#scipp.Dataset) can be indexed in a similar manner to NumPy and xarray.\n",
+    "Objects in scipp can be sliced in two ways. The general way to do this is by `\"positional indexing\"` using indices that that denote position. \n",
+    "A second approach is to use `\"location based indexing\"` which is uses actual coordinate values for selection."
```

Comment:
Previous comment not addressed:
> Where did you find the term "location indexing"? Xarray seems to call this "label-based indexing". Location indicing is easily confused with position (aren't the two synonyms?)

Reply:
That's a typo, should be `label based indexing` which is used everywhere else in this document and should be consistent with xarray and pandas.